source/extensions/filters/http/ext_proc/ext_proc.cc

```diff
@@ -331,24 +366,31 @@ void Filter::onReceiveMessage(std::unique_ptr<ProcessingResponse>&& r) {
 
   switch (response->response_case()) {
   case ProcessingResponse::ResponseCase::kRequestHeaders:
+    ENVOY_LOG(debug, "Received RequestHeaders response");
     message_handled = decoding_state_.handleHeadersResponse(response->request_headers());
     break;
   case ProcessingResponse::ResponseCase::kResponseHeaders:
+    ENVOY_LOG(debug, "Received ResponseHeaders response");
     message_handled = encoding_state_.handleHeadersResponse(response->response_headers());
     break;
   case ProcessingResponse::ResponseCase::kRequestBody:
+    ENVOY_LOG(debug, "Received RequestBody response");
     message_handled = decoding_state_.handleBodyResponse(response->request_body());
     break;
   case ProcessingResponse::ResponseCase::kResponseBody:
+    ENVOY_LOG(debug, "Received ResponseBody response");
     message_handled = encoding_state_.handleBodyResponse(response->response_body());
     break;
   case ProcessingResponse::ResponseCase::kRequestTrailers:
+    ENVOY_LOG(debug, "Received RequestTrailers response");
```

Comment:
Nit: could have a enum-to-string converter to make this less verbose.

Reply:
Is there a canonical Envoy way to do this, such as an "operator <<" or a ToString method? I didn't find anything in filter.h for the filter callback enums, for instance.

Reply:
I added a method to convert the status to string -- only using it here but in the future maybe in other places.